### DISCUSSION TITLE
beans 表怎么只能配置一个类啊？ 在下面继续加没用

### DISCUSSION THREAD

**Discussion by SinForLove** [focus-creative-games/luban#109] [Q&A] [answered]
郁闷，检查了一下也没写错啊
![QQ截图20240131151702](https://github.com/focus-creative-games/luban/assets/78003316/b945511c-8ab7-4e4c-9b0c-620c8018f10c)

**Comment by DustinMaple** [comment]
你把valueType那一列删了试试

**Reply by SinForLove** [reply, marked as answer]
我的错 ， luban貌似会把beans 里声明的但是没有在其它表里引用的类型默认优化掉了
